Menu/Menu.js

```diff
@@ -1,10 +1,15 @@
 
 const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
+  // Toggle the "menu--open" class on your menu refence.
+  menu[0].classList.toggle('menu--open');
+  return 'awesome'
 }
 
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.getElementsByClassName('menu');
```

Comment:
if you got rid of this s you could get the element and not an array, and that way you dont need [0] in front of your elements in this file.
```suggestion
const menu = document.getElementByClassName('menu');

menu.classList.toggle('menu--open');
```